.internal-ci/README.md

```diff
@@ -36,58 +37,61 @@ v0.0.0-my-awesome-feature.21.sha-abcd1234
 
 **Release branches**
 
-- `release/v1.2.0` valid characters `v[0-9]+/.[0-9]+/.[0-9]+`
+- `release/v2` valid characters `v[0-9]+`
 - Release branches will be normalized for versioning, namespaces, dns...
-  - namespaces will be prefixed with `release-`
-  - semver will be set to match the branch name.
-  - `-dev` and full tags will be create for the artifacts.
+  - namespaces will be prefixed with `mc-`
+  - semver portion will be set to `v0`.
 
 format:
 ```
-v1.2.3-${GITHUB_RUN_NUMBER}.sha-${sha}
-v1.2.3-dev
+v0-${GITHUB_RUN_NUMBER}.sha-${sha}
 ```
 
 examples:
 ```
-release/v1.2.3
+release/v2
+
+v0-21.sha-abcd1234
+```
+
+**Tags**
 
-v1.2.3-21.sha-abcd1234
-v1.2.3-dev
+- `v2.0.0` valid characters `v[0-9]+\.[0-9]+\.[0-9]+`
+- Tags will be normalized for versioning, namespaces, dns...
+  - namespaces will be prefixed with `mc-`
+  - semver will be set to match the branch name.
+  - tags will create a `v{tag}-dev` release for use in static environments
+
+format:
+```
+v2.0.0-${GITHUB_RUN_NUMBER}.sha-${sha}
+v2.0.0-dev
 ```
 
+
+
 ## CI triggers
 
 This workflow is set up to trigger of certain branch patterns.
 
-### Feature Branches - `feature/*`
+### Branches - `feature/*, release/*, master`
 
 Feature branches will trigger a build that will create a dynamic development environment and run integration tests against the environment.
 
 | Tags | SGX_MODE | IAS_MODE | Signer | Description |
 | --- | --- | --- | --- | --- |
 | `v0.0.0-my-awesome-feature.21.sha-abcd1234` | `HW` | `DEV` | CI Signed Development | For use in development environments. |
 
-### Release Branches - `release/*`
+### Semver tags - `v2.0.0`
 
 Release branches will trigger a build that will create a set of release artifacts.
```

Comment:
:wrench: similar to above should the text be updated for _tags_
```suggestion
### Semver tags - `v2.0.0`

Tags will trigger a build that will create a set of release artifacts.
```

Reply:
Thanks!